### ISSUE TITLE
PreheatIntent and CoolingIntent doesn't check return status

### ISSUE THREAD

**Issue by filcole** [ScottHelme/AlexaNissanLeaf#9]
The remote turn on of climate control may fail if the battery is too low.  This should be checked for and the user informed if the battery is too low.  I think for an example of the response see here: https://github.com/joshperry/carwings/blob/master/protocol.markdown#hvac-remote-activate-result

**Comment by ScottHelme** [comment]
Yeah, the current request is 'fire and forget' by design. Checking for the result generally takes a fair amount of time, during which Alexa would just be sat there and could possibly time out. I think a better approach would be a new intent to check if the AC is on or not, so that can be done separately if the user wants to check it succeeded or not.
